documentation/gradle/migrate.md

```diff
@@ -361,8 +361,9 @@ flyway {
 }
 ```
 
-<h2>Sample output</h2>
-<pre class="console">&gt; gradle flywayMigrate -i
+## Sample output
+
+<pre class="console">> gradle flywayMigrate -i
```

Comment:
I assume this doesn't change the HTML output?

Reply:
Indeed.